lib/ores_api.rb

```diff
@@ -20,17 +20,22 @@ def self.valid_wiki?(wiki)
     wiki.project == 'wikipedia' && AVAILABLE_WIKIPEDIAS.include?(wiki.language)
   end
 
-  def initialize(wiki)
+  def initialize(wiki, course = nil)
     raise InvalidProjectError unless OresApi.valid_wiki?(wiki)
     @project_code = wiki.project == 'wikidata' ? 'wikidata' + 'wiki' : wiki.language + 'wiki'
     @project_model = wiki.project == 'wikidata' ? 'itemquality' : 'articlequality'
+    @course = course
   end
 
   def get_revision_data(rev_ids)
-    response = ores_server.get query_url(rev_ids)
+    server_conn = ores_server
+    url_prefix = server_conn.url_prefix
```

Comment:
what's this?

Reply:
What this is doing is extracting the prefix of the url of `Faraday::Connection` object. I couldn't really make sense of how to get the full url like `https://ores.wikimedia.org/v3/scores/wikidatawiki/?models=itemquality&features&revids=1136533131|1136536523` . The `server_conn.url_prefix.to_s` in this case is basically `https://ores.wikimedia.org/` and the `url_query` part is `/v3/scores/wikidatawiki/?models=itemquality&features&revids=1136533131|1136536523`. I had to use these variables in the recue block below to pass on the full url to `save_course_error_record`.

Reply:
I also have the same doubt in `def mediawiki` in `WikiApi` on what is the correct way to get the url of the `MediawikiApi::Client` object `@mediawiki`

Reply:
Since this prefix is the same thing that you pass when setting up the Faraday object in the `ores_server` method, this will be simpler to just extract as a constant and re-use, without getting involved in additional methods on the connection object.

Reply:
Okay..Done!